### DISCUSSION TITLE
This user account is blocked

### DISCUSSION THREAD

**Discussion by BE-DEVELOPER-S** [vercel/community#5364] [Account Issue]
### Question

I just created my vercel and trying to connect to GitHub and then first thing I get is this user account is blocked. Please help @bernardemmanuel002@gmail.com

**Comment by Chan-Min-Thu** [comment]
I created my vercel account and when I  connect with GitHub ,there is getting error this user accont is blocked.I need your herlp , @chanminthu961549167@gmail.com

**Comment by LucasWeb3** [comment]
Same problem here, have been logged in earlier but not this... Please help @lillja@me.com
![Uploading Screenshot 2024-01-07 at 12.07.57.png…]()

**Reply by BorzovRuslanR** [reply]
The problem with registering an account in general is that you receive a registration confirmation email, and then when you try to log in it says This user account is blocked.
